tests/unit/profiles-management/helpers/test_k8s.py

```diff
@@ -0,0 +1,31 @@
+from lightkube.generic_resource import GenericNamespacedResource
+from lightkube.models.meta_v1 import ObjectMeta
+
+from profiles_management.helpers.k8s import get_annotations, to_rfc1123_compliant
+
+
+def test_annotations():
+    resource = GenericNamespacedResource(
+        metadata=ObjectMeta(name="test", annotations={"test": "value"})
+    )
+
+    assert get_annotations(resource) == {"test": "value"}
+
+
+def test_no_annotations():
+    resource = GenericNamespacedResource(metadata=ObjectMeta(name="test"))
+
+    assert get_annotations(resource) == {}
+
+
+def test_non_alphanum_to_hyphen():
+    name = "kimonas@canonical.com"
+
+    assert to_rfc1123_compliant(name) == "kimonas-canonical-com"
+
+
+def test_more_than_63_char_name():
+    name = "I-had-to-think-of-a-reeeally-long-string-to-use-for-the-test-which-was-tough"
+
+    assert len(name) > 63
```

Comment:
What's the purpose of this assertion? Does it contribute to the test?

Reply:
Wanted to make sure the string `name` is more than 63 chars, and make it a bit more apparent for future reference.

But if you feel this is a bit redundant we can remove it.

Reply:
It's not because of redundancy, it's because it's asserting something that is not part of the code to test. I suggest we remove the assertion.

Reply:
ACK! It's updated in the latest code, and the unit tests only check that the output is less than 64